source/frontend/src/features/mapSideBar/disposition/models/DispositionFormModel.ts

```diff
@@ -102,6 +93,20 @@ export class DispositionFormModel implements WithDispositionTeam {
     };
   }
 
+  private toPropertyApi(x: PropertyForm): ApiGen_Concepts_DispositionFileProperty {
```

Comment:
Would it be worth creating a single function that converts to ApiGen_Concepts_FileProperty instead?

Reply:
Yes, we could create that for the 2 files that don't have extra info in there (Disposition and Acquisition). Research and Lease I believe have extra fields but we could call the shared function and then append the extra stuff. Will look into it